Utils/API/server/lib/dkb/api/handlers.py

```diff
@@ -253,3 +253,29 @@ def task_kwsearch(path, **kwargs):
 
 
 methods.add('/task', 'kwsearch', task_kwsearch)
+
+
+def task_deriv(path, **kwargs):
+    """ Calculate statistics of derivation efficiency.
+
+    :param path: full path to the method
+    :type path: str
+    :param project: project name
+    :type project: str
+    :param amitag: amitag (or several)
+    :type amitag: str or list
+
+    :return: calculated statistics
+    :rtype: dict
+    """
+    method_name = '/task/deriv'
+    project = kwargs.get('project')
+    if project is None:
+        raise MissedArgument(method_name, 'project')
+    tags = kwargs.get('amitag')
+    if tags is None:
+        raise MissedArgument(method_name, 'amitag')
```

Comment:
Just a comment: it is OK to just check if given key is in hash `kwargs`, like `if 'project' not in kwargs: ...`. Taking exact value is useful when you need to check the value itself; if not -- then it is an excessive action.

Reply:
True. Done.